.github/workflows/publish-release-artifacts.yml

```diff
@@ -10,7 +10,8 @@ jobs:
       - name: Download Artifacts
         run: |
           echo "Release Tag: ${{ github.event.release.tag_name }}"
-          ./scripts/github/release-artifacts.sh --tag "${{ github.event.release.tag_name }}" --workflow "Release Artifacts" --repo-org "${{ github.repository_owner }}" download
+          binary_repo="mayastor-extensions"
+          ./scripts/github/release-artifacts.sh --tag "${{ github.event.release.tag_name }}" --workflow "Release Artifacts" --repo-org "${{ github.repository_owner }}" --binary-repo "$binary_repo" download
```

Comment:
Add a default value for the repo, so we don't modify the current api for existing use-cases